src/Promitor.Agents.Scraper/Validation/Steps/Sinks/OpenTelemetryCollectorMetricSinkValidationStep.cs

```diff
@@ -48,6 +72,11 @@ public ValidationResult Run()
                     errorMessages.Add($"Configured URI ({collectorUri}) for the OpenTelemetry Collector is not a valid URI.");
                 }
             }
+            // Protocol Validation
+            if (!TryParseProtocol(collectorProtocol, out var parsedProtocol))
```

Comment:
Why do we need this? If the protocol would be bad, then it would not be present. I think we can drop this logic and just rely on the SDK

Reply:
Removing